INDEX

```diff
@@ -5,222 +5,250 @@
         "icon": "https://www.trueos.org/iocage-icons/asigra.png",
         "description": "Full-featured backup solution fully integrated into FreeNAS. (Requires Subscription)",
         "primary_pkg": "dssystem",
-        "official": true
+        "official": true,
+        "category": "backup"
```

Comment:
@sirdik can we please move `category` to each plugin's manifest file ? While we are at this, it would be great if we did the same for `primary_pkg` key. This helps us save time not parsing INDEX when querying installed plugins.

Reply:
@sonicaj  Files updated. Do we want to move also the "official" parameter into each plugin's manifest? Now we have this information duplicate (and in some cases inconsistent)

Reply:
@dlavigne do we want to expose `official` option for installed plugins in the UI ? It is currently only displayed for plugins which are available to download.
@sirdik it depends, if we want to expose it in the UI, then yes we should move them to plugin manifest, else no